tests/Integration/AbstractIntegrationTestCase.php

```diff
@@ -1,13 +1,15 @@
 <?php
 
+declare(strict_types = 1);
+
 namespace DrevOps\Robo\Tests\Integration;
 
-use DrevOps\Robo\Tests\AbstractTest;
+use DrevOps\Robo\Tests\AbstractTestCase;
 
 /**
- * Class AbstractTest
+ * Class AbstractTestCase
  */
-abstract class AbstractIntegrationTest extends AbstractTest
+abstract class AbstractIntegrationTestCase extends AbstractTestCase
```

Comment:
Because I update phpunit to 10, we need to create abstract class without suffix Test.